services/getUserByRole.js

```diff
@@ -0,0 +1,37 @@
+const {ddbClient} = require('../dbClient/ddbClient');
+const { TABLE_NAME, GSI } = require('../shared/constants');
+
+const getUserByRole = (role) => {
+    const params = {
+        TableName: TABLE_NAME,
+        IndexName: GSI,
+        ExpressionAttributeValues: {
+            ':role': role
+        },
+        ExpressionAttributeNames: {
+            '#r': 'role'
+        },
+        KeyConditionExpression: '#r = :role'
+    }
+
+    return ddbClient.query(params).promise();
```

Comment:
Query API can return `LastEvaluateKey` in case the data exceed maximum return bytes from dynamoDB

https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Query.Pagination.html